python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vlans.schema.yml

```diff
@@ -23,6 +23,21 @@ keys:
         state:
           type: str
           valid_values: ["active", "suspend"]
+        address_locking:
+          type: dict
+          keys:
+            address_family:
```

Comment:
I think PR description and heading should be updated if we are adding `address_family` as well

Reply:
updated